_install-and-configure/install-opensearch/debian.md

```diff
@@ -136,14 +138,27 @@ APT, the primary package management tool for Debian–based operating systems, a
 
 1. Choose the version of OpenSearch you want to install: 
    - Unless otherwise indicated, the latest available version of OpenSearch is installed.
+
    ```bash
+   # For new installations of OpenSearch 2.12 and later, you must define a custom admin password in order to set up a security demo configuration.
+   # Use one of the following commands to define a custom admin password:
```

Comment:
Global: Please confirm proper formatting (Should there be so many `#` in the examples?).